routes/index.js

```diff
@@ -32,7 +32,7 @@ const UPLOAD_QUERY_LIMIT = 2500;
 const multerOptionsFasta = {
   dest: 'upfasta/',
   limits: {
-    fileSize: 1000000 //1mb
+    fileSize: 5000000 //5mb
```

Comment:
Can you increase it to 10mb? We decided it in the last meeting to increase it to 10 based on Dan's 500 sequences.